web/styles/settings.css

```diff
@@ -193,11 +193,6 @@ h3,
     }
 }
 
-td .button {
-    margin: 2px 0;
-    box-shadow: none;
-}
```

Comment:
Is the story with these deleted blocks that the `new-style` copy was taking precedence?

Reply:
Confirmed that is the case.